### PR TITLE
Added Candu doc and image

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/testing-and-personalization/candu.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+          promotion_id: 'promo1',
+          creative: 'banner1',
+          name: 'sale',
+          position: 'home_top'
+      });
+```
+
+<div class="warningBlock">
+
+While working with Candu platform, the `identify` call should always be made before the `track` call. The reason being if a `track` call is made first for an anonymous user, then a user is created with anonymousId. Further, when an `identify` call is made for the same user where the user has userId and traits, a separate user is created in Candu. Hence, two identities are created for the same user.
```
Comment (utsabc) [inline destinations/testing-and-personalization/candu.mdx:91]: Can we rephrase this a bit?
"While working with Candu platform it is recommended to make an `identify` call before making a `track` call to ensure no duplicate identities are created in Candu platform"

File: destinations/testing-and-personalization/candu.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+## Connection settings
+
+To successfully configure Candu as a destination, you will need to configure the following settings:
+
+- **API Key**: Your API Key is the unique key generated against your Candu account. It can be found in the **Settings** option, under **Workspaces** tab, in the **Access Keys** section of your Candu account. Refer [FAQ section](#faq) for more details.
+
+
+## Identify
+
+The identify call lets you capture the actions of a visiting user. It also lets you record any traits associated with them such as their name, email address, etc.
```
Comment (utsabc) [inline destinations/testing-and-personalization/candu.mdx:54]: Just a minor change
"The identify call lets you capture the details of a visiting user. It lets you record any traits associated with them such as their name, email address, etc."

**Review by utsabc — CHANGES_REQUESTED** [review]
Some minor changes requested

**Comment by ishachopra0401** [comment]
@utsabc Suggested changes have been implemented. Please have a look.

File: destinations/testing-and-personalization/candu.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+</div>
+
+## Getting started
+
+Before configuring Candu as a destination in RudderStack, verify if the source platform is supported by Candu by referring to the table below:
+
+| **Connection Mode** | **Web**       | **Mobile**    | **Server**    |
+| :------------------ | :------------ | :------------ | :------------ |
+| **Device mode**     | -             | -             | -             |
+| **Cloud mode**      | **Supported** | **Supported** | **Supported** |
```
Comment (saikumarrs) [inline destinations/testing-and-personalization/candu.mdx:26]: @utsabc @ItsSudip we should check if it makes sense for these platforms to be a source of LaunchDarkly event data.

**Review by utsabc — APPROVED** [review]

File: destinations/testing-and-personalization/candu.mdx (hunk 3 above)
Comment (utsabc) [reply destinations/testing-and-personalization/candu.mdx:26]: Yep they have SDK for we and mobile we can support them in future releases

Comment (ItsSudip) [reply destinations/testing-and-personalization/candu.mdx:26]: I don't see any mobile sdk for candu, we should update the documentation with web support only.

Comment (saikumarrs) [reply destinations/testing-and-personalization/candu.mdx:26]: No. The discussion is here only about the "cloud mode" row.

**Review by saikumarrs — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**